openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -113,6 +113,114 @@
   border-radius: 9999px;
 }
 
+// Width
+.w-4 {
+  width: 16px;
+}
+.w-5 {
+  width: 24px;
+}
+.w-8 {
+  width: 32px;
+}
+.w-9 {
+  width: 36px;
+}
+.w-16 {
+  width: 64px;
+}
+.w-24 {
+  width: 6rem;
+}
+.w-32 {
+  width: 8rem;
+}
+.w-36 {
+  width: 9rem;
+}
+.w-72 {
+  width: 288px;
```

Comment:
aren't we already having this class?

Reply:
@ShaileshParmar11  we are having, feel like due to the main merge and change in the position they are showing.